qlib/data/dataset/handler.py

```diff
@@ -71,38 +71,58 @@ def __init__(
             start_time of the original data.
         end_time :
             end_time of the original data.
-        freq :
-            frequency of data
         data_loader : Tuple[dict, str, DataLoader]
             data loader to load the data.
         init_data :
             intialize the original data in the constructor.
         fetch_orig : bool
             Return the original data instead of copy if possible.
+        **kwargs:
+            it will be passed into data_loader
         """
         # Set logger
         self.logger = get_module_logger("DataHandler")
 
         # Setup data loader
         assert data_loader is not None  # to make start_time end_time could have None default value
 
+        # what data source to load data
         self.data_loader = init_instance_by_config(
             data_loader,
             None if (isinstance(data_loader, dict) and "module_path" in data_loader) else data_loader_module,
             accept_types=DataLoader,
+            **kwargs,
```

Comment:
Is it necessary to pass kwargs to `data_loader`?
I think all arguments of `data_loader` should be in config.

Reply:
Is handlers in contrib should keep consistent with example? Pass arguments to data_loader with kwargs or config?